extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -197,8 +197,11 @@ export class Bind {
       Object.keys(parseErrors).forEach(expressionString => {
         const elements = this.expressionToElements_[expressionString];
         if (elements.length > 0) {
-          const err = user().createError(parseErrors[expressionString]);
-          reportError(err, elements[0]);
+          const parseError = parseErrors[expressionString];
+          const error = new Error(parseError.message);
+          error.stack = parseError.stack;
+          const userError = user().createError(error);
```

Comment:
Nit: You can pass `parseError.message` directly into `#createError`, and set `#stack` on the returned error.